modules/hpe_nimble_acr.py

```diff
@@ -0,0 +1,297 @@
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+author:
+    - Alok Ranjan (@ranjanal)
+description: "On HPE Nimble Storage Array - Create or delete ACL for volume"
+module: hpe_nimble_acr
+
+options:
+    state:
+        required: True
+        choices:
+            - present
+            - absent
+            - create
+        type: 'str'
+        description:
+        - Choice for ACR operation
+    apply_to:
+        required: False
+        choices:
+            - volume
+            - snapshot
+            - both
+            - pe
+            - vvol_volume
+            - vvol_snapshot
+        type: str
+        default: both
+        description:
+        - Type of object this access control record applies to.
+    chap_user:
+        required: False
+        type: str
+        description:
+        - Name for the CHAP user.
+    lun:
+        required: False
+        type: int
+        description:
+        - If this access control record applies to a regular volume, this attribute is the volume's LUN (Logical Unit Number).
+          If the access protocol is iSCSI, the LUN will be 0. However, if the access protocol is Fibre Channel, the LUN will be in the range from 0 to 2047
+    volume:
+        required: False
+        type: str
+        description:
+        - Name for the volume this access control record applies to.
+    pe_ids:
+        required: False
+        type: list
+        description:
+        - List of candidate protocol endpoints that may be used to access the Virtual Volume. One of them will be selected for the access control record.
+          This field is required only when creating an access control record for a Virtual Volume.
+    protocol_endpoint:
+        required: False
+        type: str
+        description:
+        - Name for the protocol endpoint this access control record applies to.
+    snapshot:
+        required: False
+        type: str
+        description:
+        - Name of the snapshot this access control record applies to.
+    initiator_group:
+        required: False
+        type: str
+        description:
+        - The Initiator Group name.
+
+extends_documentation_fragment: hpe_nimble
+short_description: "Manages a HPE Nimble Storage ACR"
+version_added: "2.9"
+'''
+
+EXAMPLES = r'''
+
+    # if state is create, create ACL for given volume, fails if it exist or cannot create
+    # if state is present, create ACL if not present, else success
+    - name: Create ACR for volume
+      hpe_nimble_acr:
+        hostname: "{{ hostname }}"
+        username: "{{ username }}"
+        password: "{{ password }}"
+        volume: "{{ volume }}"
+        initiator_group: "{{ initiator_group }}"
+        state: "{{ state | default('present') }}" # fail if exist
+
+    # Delete the ACL for a given volume name
+    - name: Delete ACR for volume
```

Comment:
Volume?

Reply:
as mentioned in volume module.
I have switched from Volume to "volume" for consistency. Is that fine?